test/subset/run-tests.py

```diff
@@ -77,6 +78,7 @@ def run_test(test, should_check_ots):
 	return 0
 
 def run_ttx(file):
+	print ("ttx %s" % file)
```

Comment:
I think it would be fine to print the command for all cmd() calls; always useful when troubleshooting

Reply:
Good idea, I'll do that separate from this pr.